.github/workflows/docs.yml

```diff
@@ -0,0 +1,55 @@
+name: Documentation CI
+on:
+  pull_request:
+    paths:
+      - .vale.ini
+      - docs/**/*.md
+      - Library/Homebrew/**/*.rb
+
+permissions:
+  contents: read
+
+jobs:
+  docs:
+    if: github.repository == 'Homebrew/brew'
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: docs
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Run vale for docs linting
+        run: |
+          brew install vale
```

Comment:
Might be nicer to install this in a separate step to differentiate "you broke `brew`" errors.